src/AddTodoForm.js

```diff
@@ -1,10 +1,21 @@
 import React from 'react';
 
-const AddTodoForm = () => {
+const AddTodoForm = (props) => {
+  const handleAddTodo = (event) => {
+    event.preventDefault();
+    const form = event.target;
+    const title = document.getElementById('todoTitle');
+    const todoTitle = title.value;
+    console.log(todoTitle);
+    props.onAddTodo(todoTitle);
+    form.reset();
```

Comment:
Nice job figuring out how to use `reset()` as well as assigning the form element to its own variable `form` . This does help with clarity. 
Something to consider. What if you have multiple form elements in the component? How would you adjust the name?